static/bidder-info/grid.yaml

```diff
@@ -5,3 +5,7 @@ capabilities:
     mediaTypes:
       - banner
       - video
+  app:
+    mediaTypes:
+      - banner
+      - video
```

Comment:
Nitpick: can `app` come first in the list please?
```
capabilities:
  app:
    mediaTypes:
      - banner
      - video
  site:
    mediaTypes:
      - banner
      - video
```

Reply:
Done